liquid_node/commands.py

```diff
@@ -337,6 +337,12 @@ def purge(force=False):
         print('No collections will be purged')
 
 
+def deletecollection(name):
+    """Delete a collection by name"""
+    nomad.stop(f'collection-{name}')
+    purge_collection(name)
```

Comment:
This will not remove it from the ini file

Reply:
And it should not. I see two scenarios:
1. Really remove a collection - run `deletecollection` and remove it from the ini file, in any order.
2. Reprocess a collection from scratch, useful in development and testing - run `deletecollection` and then `deploy`.

I'm mostly interested in the 2nd scenario, and with this command, iterating is pretty smooth.